xray/services/scan_test.go

```diff
@@ -47,3 +49,32 @@ func TestCreateScanGraphQueryParams(t *testing.T) {
 		})
 	}
 }
+
+func TestFlattenGraph(t *testing.T) {
```

Comment:
Let's add another test to check no duplications

Reply:
Already checked it in the test (with the set)